source/release-notes/1.9.txt

```diff
@@ -22,6 +22,16 @@ This page describes changes and new features introduced in
 
 **Upcoming**
 
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.9.0, ``mongosync`` includes an embedded verifier to
```

Comment:
Can we link to the verifier page https://deploy-preview-339--docs-cluster-to-cluster-sync.netlify.app/reference/verification/embedded/?

Reply:
Fixed